README.md

```diff
@@ -251,6 +251,18 @@ Now that you have created two tables, you'll see entries for them in the `pointc
 >     INSERT INTO patches (pa)
 >     SELECT PC_Patch(pt) FROM points GROUP BY id/10;
 
+
+**PC_Patch(pcid integer, vals float8[])** returns **pcpatch**
```

Comment:
`PC_MakePoint`

Reply:
I think you meant `PC_MakePatch`, but your greater point stands. :)